Documentation/configuration/metrics.rst

```diff
@@ -98,3 +98,12 @@ It is also possible to hard-code ``static-config`` sections that are simply an I
             labels:
               node-id: i-0598c7d7d356eba47
               node-az: a
+
+Prometheus-operator Integration
+===============================
+To integrate with `Prometheus-operator <https://coreos.com/operators/prometheus/docs/latest/>`_, create a Service object (https://github.com/cilium/cilium/blob/master/examples/kubernetes/cilium-metrics-svc.yaml) and ServiceMonitor (https://github.com/cilium/cilium/blob/metrics/examples/kubernetes/prometheus-k8s-service-monitor-cilium.yaml).
```

Comment:
Should we link to 'latest' from Prometheus docs? Or should we link to a specific version?

Reply:
I wanted to put https://coreos.com/operators/prometheus/. 
But this redirects to the latest docs!
I will use this link and not worry about the redirect.

Reply:
@abhide15 https://coreos.com/operators/prometheus/docs/0.17.0/index.html works fine